src/main/scala/org/allenai/deep_qa/experiments/datasets/Science.scala

```diff
@@ -83,6 +136,51 @@ object ScienceDatasets {
     ScienceCorpora.buscElasticSearchIndex(3)
   )
 
+  /*
+   * Fourth Grade Questions in RC Format (BUSC-fetched passage,
+   * question, answer options)
+   */
+  val readingComprehensionOmnibusQaGradeFourTrainQuestionsWithBuscBackground: JValue =
```

Comment:
To follow the same convention as above, this should be named `omnibusRcGradeFourTrainQuestionsWithBuscBackground`.